_pkgdown.yml

```diff
@@ -1,277 +1,281 @@
+      - env_inherits
+      - is_environment
+      - is_bare_environment
+      - as_environment
+      - caller_env
+  - title: Environment bindings
+    contents:
+      - env_bind
+      - env_bind_lazy
+      - env_bind_active
+      - env_unbind
+      - with_bindings
+      - scoped_bindings
+      - env_has
+      - env_get
+      - env_get_list
+      - env_names
+      - env_length
+      - env_poke
+  - title: Search path and namespaces
+    contents:
+      - search_envs
+      - search_env
+      - base_env
+      - global_env
+      - empty_env
+      - is_installed
+      - is_namespace
+      - ns_env
+      - pkg_env
+      - env_name
+      - env_label
+  - title: S3
+    contents:
+      - inherits_any
+      - inherits_all
+      - inherits_only
+      - new_box
+      - is_box
+      - as_box
+      - as_box_if
+      - unbox
+      - zap
+      - is_zap
+      - done
+  - title: Create vectors
+    desc: >
+      Atomic vector constructors have implicit splicing (i.e. splice
+      lists automatically), while the list constructor has explicit
+      splicing.
+    contents:
+      - lgl
+      - int
+      - dbl
+      - cpl
+      - chr
+      - list2
+      - rep_along
+      - rep_named
+      - seq2
+      - seq2_along
+  - title: Attributes
+    contents:
+      - set_names
+      - names2
+      - has_name
+      - is_named
+  - title: Type predicates
+    contents:
+      - is_list
+      - is_atomic
+      - is_vector
+      - is_integer
+      - is_double
+      - is_character
+      - is_logical
+      - is_raw
+      - is_bytes
+      - is_scalar_list
+      - is_scalar_atomic
+      - is_scalar_vector
+      - is_scalar_integer
+      - is_scalar_double
+      - is_scalar_character
+      - is_scalar_logical
+      - is_scalar_raw
+      - is_scalar_bytes
+      - is_bare_list
+      - is_bare_atomic
+      - is_bare_vector
+      - is_bare_double
+      - is_bare_integer
+      - is_bare_numeric
+      - is_bare_character
+      - is_bare_logical
+      - is_bare_raw
+      - is_bare_bytes
+      - is_empty
+      - is_integerish
+      - is_bare_integerish
+      - is_scalar_integerish
+      - is_null
+      - is_true
+      - is_false
+      - is_string
+      - is_bool
+      - is_bare_string
+  - title: Operators
+    contents:
+      - "`%||%`"
+      - "`%|%`"
+      - "`%@%`"
+  - title: Function arguments
+    contents:
+      - arg_match
+      - missing_arg
+      - is_missing
+      - maybe_missing
+  - title: Create tidy evaluation APIs
+    desc: >
+      Create data masking APIs à-la dplyr or tidyr.
+      These functions are meant for developers rather than users.
+    contents:
+      - "`tidyeval-data`"
+      - eval_tidy
+      - as_data_mask
+      - new_data_mask
+      - as_data_pronoun
+  - title: Condition handlers
+    contents:
+      - with_handlers
+      - calling
+  - title: Session state
+    contents:
+      - matches("interactive")
+      - matches("^[^(faq)]option")

```

Comment:
That regex doesn't actually do what I want. I think the `matches("option")` should be changed to `ends_with("option")` to pick up all of the options documentation without faq-options. Unless we want the global options to be documented under both session state and FAQ

Reply:
Yes it seems fine to document in both sections.

Can you double check your changes to the pkgdown file please? It seems you have reindented everything. The commit should not contain any whitespace changes.

Reply:
All fixed up, sorry about that.